.circleci/scripts/cherry-picker.sh

```diff
@@ -124,6 +145,18 @@ if [[ "$ret" -ne 0 ]]; then
     exit 0
 fi
 
+# Attach label for latest release branch if 'docs-cherrypick' is present. Will noop if already applied.
+latest_backport_label=$(get_latest_backport_label)
+status "latest backport label is $latest_backport_label"
+if echo "$resp" | jq -e '.items[].labels[] | select(.name | contains("docs-cherrypick"))'; then
+    labels=$(curl -f -s -H "Authorization: token ${GITHUB_TOKEN}" -X POST -d "{\"labels\":[\"$latest_backport_label\"]}" "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/issues/${pr_number}/labels" | jq --raw-output '.[].name' | sort -rV)
```

Comment:
Do you know if this will apply the label synchronously/if GitHub's API has "read your own writes" consistency? Basically curious if we should _explicitly_ trigger the backport flow by setting an env var here or something in addition to applying the label.

Reply:
I can't find docs one way or another but if it doesn't return with a 200 it will error check out of this `if` and fail to apply at all. 

I tried to do this a few ways including appending the backport label to the variable holding the `$labels` but I thought that would be too brittle messing with the API response. I decided to use the POST response instead because irrespective of whether the label is added immediately or queued in the backend, the response should contain an array of all labels + the backport one I added. This will trigger the cherrypicking on the backport branch regardless of if the label actually applied which is the more important case to handle. Functionally, seeing the label will definitely make it easier to filter quickly whether something _should_ be cherrypicked but the real signal is in the PR comment of cherrypick success/failure.

Reply:
Oh that's perfect, didn't realize the POST response would capture it!